app/assets/stylesheets/partials/_header.css.scss

```diff
@@ -0,0 +1,20 @@
+header.navbar {
+  background-color: #232120;
```

Comment:
Color literals like `#232120` should only be used in variable declarations; they should be referred to via variable everywhere else.